docs/clusterobservability.md

```diff
@@ -166,19 +166,77 @@ The various storage configuration per capability can be achieved by multiple `Cl
 ### Object storage types
 
 Each object storage type has its own set of required fields which are configured in the secret.
-Object storage types will be added separately, there are plans to support the all the storage types supported by the capabilities.
+There are plans to support all storage types required by the capabilities.
```

Comment:
Putting this here because I can only comment on changed lines:

> The storage section of the `ClusterObservability` CRD allows users to configure the storage for all supported observability backends.

If I am not misunderstanding this, then this approach seems to assume that you can put all three signals into a single bucket side-by-side. Have we tested this? For example, I know that Loki is currently not able to run with a prefix (subdirectory) inside a bucket.

Reply:
That is a great comment, I will validate it. However, if there are issues there is still value in having a common configuration API for all capabilities.

Reply:
+1 - perhaps we need a common config to provide a default S3 bucket, but allow each backend to partially or completely override the default settings. E.g. one option would be to have separate buckets per back-end, another might be to use entirely separate S3 stores.